### PR TITLE
login-page-design

### LINKED ISSUE #100: Match the login to the design
### Actual
chrome_VNX59Dt0M1

### Expected
chrome_MNpx2aaU24

### Ref
https://xd.adobe.com/view/63af58cd-fa23-4cf0-a7bc-1f5fc1774b8e-36fc/screen/14c32bbd-d656-473d-b19b-704675b147e7/

### PR DESCRIPTION
**Pull request by edgardorms**
modificacion de login page

### CODE REVIEW

**Comment by angelik0828** [comment]
## ✔️  Checked and found improvement on text fields, they no longer appear blank

image

**Review by angelik0828 — APPROVED** [review]

File: src/pages/Login/Login.layout.vue [hunk 1]
```diff
@@ -16,9 +16,10 @@
       >
         <q-input
           input-class="login__input"
-          rounded
           color="black"
           outlined
+          autofocus
```
Comment (Eghinner) [inline src/pages/Login/Login.layout.vue]: better not

File: src/components/HeaderWithSearchBar/styles.scss [hunk 2]
```diff
@@ -47,7 +47,7 @@
 
   &__input {
     .q-field__control {
-      height: 2.8125rem;
+      height: 3rem;
```
Comment (Eghinner) [inline src/components/HeaderWithSearchBar/styles.scss:50]: good here, but the width doesn't seems to be the correct

**Review by Eghinner — CHANGES_REQUESTED** [review]

File: src/pages/Login/Login.layout.vue (hunk 1 above)
Comment (edgardorms) [reply src/pages/Login/Login.layout.vue]: i erase autofocus in the new commit

File: src/components/HeaderWithSearchBar/styles.scss (hunk 2 above)
Comment (edgardorms) [reply src/components/HeaderWithSearchBar/styles.scss:50]: i make changes in the width in my new commit

File: src/pages/Login/Login.scss [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 .login {
   padding: 6rem 1.25rem 0;
+  width: 17.5rem;
```
Comment (Eghinner) [inline src/pages/Login/Login.scss]: ### Expected
chrome_WREFTTjPyU
### Actual
chrome_zOSdvEYygs

**Review by Eghinner — CHANGES_REQUESTED** [review]

**Review by Eghinner — APPROVED** [review]

**Pull request merged**